wandb/proto/wandb_internal.proto

```diff
@@ -47,6 +47,8 @@ message Record {
     LinkArtifactRecord link_artifact = 24;
     UseArtifactRecord use_artifact = 25;
     LaunchWandbConfigParametersRecord wandb_config_parameters = 26;
+    LaunchConfigFileParameterRecord config_file_parameter = 27;
```

Comment:
what is the difference could you theoretically reuse it?

Reply:
One has a file path associated with it and one does not- you can think of `LaunchWandbConfigParametersRecord` as a special case I guess. we could use a special key in `relpath` to indicate that a record applies to the `wandb.config`